packages/orchestrator/lib/events.ts

```diff
@@ -23,10 +23,11 @@ export class EventsHandler extends EventEmitter {
         this.onCallbacks = {
             CREATED: (task: Task) => {
                 on.CREATED(task);
-                this.emit(`task:started:${task.groupKey}`, task);
+                this.emit(`task:created:${task.groupKey}`, task);
```

Comment:
oops, after 15 years I still can't understand how we can read the same lines over and over, and still miss stuff like this ahah Brain is goofy